build/cmake/init.cmake

```
@@ -36,6 +36,26 @@ if(MSVC)
               "Flags used by the CXX compiler during ${cfg_upper} builds." FORCE)
         endif()
     endforeach()
+    
+    if(wxUSE_OPTIMISE)
+        set(MVC_PREFERRED_RELEASE_FLAGS " /LTCG /OPT:REF,ICF /DEBUG")
+        wx_string_append(CMAKE_EXE_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
+        wx_string_append(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${MVC_PREFERRED_RELEASE_FLAGS}")
+        wx_string_append(CMAKE_CXX_FLAGS_RELEASE " -DNDEBUG /Ox /Ob2 /Oi /Ot /Oy /GS- /Gy /GR- /GL /Gw")
```

```suggestion
        wx_string_append(CMAKE_CXX_FLAGS_RELEASE " /Ox /Oi /Ot /Oy /GS- /Gy /GL /Gw")
```
`-DNDEBUG` and `/Ob2` are enabled by default.
Can't use `/GR-`, wxWidgets code makes use of `typeid` and/or `dynamic_cast`.

Also add them for `CMAKE_C_FLAGS_RELEASE`.

---

about _/GR-_ - just a habit. Done